src/odb/src/lefin/lefin.cpp

```diff
@@ -1889,33 +1884,61 @@ void lefinReader::units(LefParser::lefiUnits* unit)
   }
 }
 
-void lefinReader::setDBUPerMicron(int dbu)
+namespace {
+bool isValidDBUPerMicron(int dbu)
 {
   switch (dbu) {
-    case 100:
-    case 200:
-    case 400:
-    case 800:
     case 1000:
```

Comment:
While not common these values are not invalid.  Why remove them?
<img width="837" height="551" alt="image" src="https://github.com/user-attachments/assets/4bed5483-e2ab-46df-b49c-c49709fdbdac" />

Reply:
@maliberty If you check the old code, lefinReader::setDBUPerMicron would never get a value less than a 1000. It's just dead code.